lib/dangermattic/plugins/manifest_pr_checker.rb

```diff
@@ -87,20 +84,46 @@ def check_swift_package_resolved_updated(report_type: :warning)
       )
     end
 
+    # Check if the `Package.swift` file was modified without a corresponding `Package.resolved` update,
+    # checking for exact path matches
+    #
+    # @param report_type [Symbol] (optional) The type of report for the message. Types: :error, :warning (default), :message.
+    #
+    # @return [void]
+    def check_swift_package_resolved_updated_strict(manifest_path:, manifest_lock_path:, report_type: :warning)
+      check_manifest_lock_updated_strict(
+        manifest_path: manifest_path,
+        manifest_lock_path: manifest_lock_path,
+        instruction: 'Please resolve the Swift packages in Xcode',
```

Comment:
If one passes `manifest_path: 'Package.swift', manifest_lock_path: 'Package.resolved'` because they are not using an Xcode project but only a bare Swift package (e.g. `Gravatar-SDK-iOS`), then it doesn't make sense to have the instruction be "Please resolve the Swift packages in Xcode" 🙃

Reply:
Ah, yes 🤦‍♂️  good catch!

Reply:
Updated on 5590610a803f86f98648effaea8b4395d2f6ef3c